File: _etc/Java/11723 집합.java
import java.util.*;

public class Main {
    
    static ArrayList<Integer> set = new ArrayList<Integer>();
    static StringBuilder sb = new StringBuilder();
    static void Add(int x) {
        if(!set.contains(x)) {
            set.add(x);
        }
    }
    static void Remove(int x) {
        if(set.contains(x)) set.remove(Integer.valueOf(x));
    }
    static void Check(int x) {
        if(set.contains(x)) sb.append("1\n");
        else sb.append("0\n");
    }
    static void Toggle(int x) {
        if(set.contains(x)) set.remove(Integer.valueOf(x));
        else set.add(x);
    }
    static void All() {
        set.clear();
        for(int i = 1; i <= 20; i++) {
            set.add(i);
        }
    }
    static void Empty() {
        set.clear();
    }
    
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
      
        while(n-- >= 0) {
            String[] order = sc.nextLine().split(" ");
            if(order[0].equals("add")) {
                Add(Integer.parseInt(order[1]));
            }
            else if(order[0].equals("remove")) {
                Remove(Integer.parseInt(order[1]));
            }
            else if(order[0].equals("check")) {
                Check(Integer.parseInt(order[1]));
            }
            else if(order[0].equals("toggle")) {
                Toggle(Integer.parseInt(order[1]));
            }
            else if(order[0].equals("all")) {
                All();
            }
            else if(order[0].equals("empty")) {
                Empty();
            }
        }
        System.out.print(sb);
    }
}

File: _etc/Kotlin/11723 집합.kt
import java.util.*;


var set = ArrayList<Int>();
var sb = StringBuilder();

fun add(x : Int) {
    if(!set.contains(x)) set.add(x);
}
fun remove(x : Int) {
    if(set.contains(x)) set.remove(x);
}
fun check(x: Int) {
    if(!set.contains(x)) sb.append("0\n");
    else sb.append("1\n");
}
fun toggle(x: Int) {
    if(set.contains(x)) set.remove(x);
    else set.add(x);
}
fun all() {
    set.clear() 
    for(i in 1..20) set.add(i);
}
fun empty() {
    set.clear()
}
fun main(args: Array<String>) {
    
    var sc = Scanner(System.`in`)
    var n = sc.nextInt()
    
    while(n-- >= 0) {
        var order = sc.nextLine().split(' ')
        
        if(order[0] == "add") {add(order[1].toInt())}
        if(order[0] == "remove") {remove(order[1].toInt())}
        if(order[0] == "check") {check(order[1].toInt())}
        if(order[0] == "toggle") {toggle(order[1].toInt())}
        if(order[0] == "all") {all()}
        if(order[0] == "empty") {empty()}
    }
    
    print(sb);
}
